usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -217,19 +215,18 @@ def get_sum_and_count_aggregation_results(keyword):
                  }
               }
               }, "size": 0}
-    found_result = 0
-    while not found_result > 10:
-        found_result += 1
+    response = es_client_query(index=index_name, body=query, retries=10)
+    if response:
         try:
-            response = CLIENT.search(index=index_name, body=query)
             results = {}
             results["prime_awards_count"] = response['aggregations']["prime_awards_count"]["value"]
             results["prime_awards_obligation_amount"] = \
                 round(response['aggregations']["prime_awards_obligation_amount"]["value"], 2)
             return results
-        except (TransportError, ConnectionError) as e:
-            logger.error(e)
-            logger.error('Error retrieving ids. Retrying connection.')
+        except KeyError:
+            logger.error('Unexpected Response')
+    else:
+        return None
```

Comment:
There should be a check in search.py to handle this return value and provide the correct exception